product_dimension/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Product Dimension
+=================
+Add dimensions (length, width and height) to products. Find the volume
+automatically when you change one of these dimensions.
+
+This module was previously hosted on github.com/ingadhoc/odoo-addons and
```

Comment:
Using the full URL https://github.com/adhoc-dev/odoo-addons would make the link clickable.